stock_picking_group_by_partner_by_carrier/models/stock_move.py

```diff
@@ -26,34 +28,40 @@ def _assign_picking_post_process(self, new=False):
             self._on_assign_picking_message_link()
         return res
 
-    def _prepare_merge_group_values(self, sales):
+    def _prepare_merge_group_values(self, move_groups):
+        sales = move_groups.sale_id
```

Comment:
mapped?

Reply:
no longer necessary on relations in 13.0 :)